app/locale/en.js

```diff
@@ -4566,7 +4566,8 @@ module.exports = {
       no_modifiers_changed_unlocks_skipped: 'It appears you attempted to unlock levels that aren\'t currently locked. Please note, if your selected level appears as locked, it might be due to a preceding level being locked. To proceed, ensure to unlock the prior level first.',
       view_custom_league: 'View Custom League',
       view_global_league: 'View Global League',
-      refresh_progress: 'Refresh Progress'
+      refresh_progress: 'Refresh Progress',
+      assign_content_first: 'Content Not Assigned! Please assign the content to students before modifying access levels.',
```

Comment:
Can we rewrite it to:
`Please ensure that <chapter/course name> is assigned to students before adjusting access levels`
If chapter/course name is difficult to get in that flow, just use `content` as string